benchmarks/build.gradle

```diff
@@ -0,0 +1,105 @@
+apply plugin: 'kotlin-multiplatform'
+apply plugin: 'kotlin-allopen'
+apply plugin: 'kotlinx.benchmark'
+
+// how to apply plugin to a specific source set?
+allOpen {
+    annotation("org.openjdk.jmh.annotations.State")
+}
+
+kotlin {
+    infra {
+        target('macosX64')
+        target('linuxX64')
+        target('mingwX64')
+    }
+
+    jvm {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = '1.8'
+            }
+        }
+    }
+
+    js {
+        nodejs()
+    }
+
+    sourceSets.all {
+        kotlin.srcDirs = ["$it.name/src"]
+        resources.srcDirs = ["$it.name/resources"]
+        languageSettings {
+            progressiveMode = true
+            useExperimentalAnnotation("kotlin.Experimental")
+            useExperimentalAnnotation('kotlin.ExperimentalUnsignedTypes')
```

Comment:
The library uses experimental unsigned types in function signatures, but doesn't propagate that experimental status on these functions.